api/types/types.go

```diff
@@ -63,3 +64,56 @@ type Code struct {
 	Language string   `bson:"language" json:"language"`
 	Files    []string `bson:"files" json:"files"`
 }
+
+// HuskyCIVulnerability is the struct that stores vulnerability information.
+type HuskyCIVulnerability struct {
+	Language       string `bson:"language" json:"language,omitempty"`
+	SecurityTool   string `bson:"securitytool" json:"securitytool,omitempty"`
+	Severity       string `bson:"severity,omitempty" json:"severity,omitempty"`
+	Confidence     string `bson:"confidence,omitempty" json:"confidence,omitempty"`
+	File           string `bson:"file,omitempty" json:"file,omitempty"`
+	Line           string `bson:"line,omitempty" json:"line,omitempty"`
+	Code           string `bson:"code,omitempty" json:"code,omitempty"`
+	Details        string `bson:"details" json:"details,omitempty"`
+	Type           string `bson:"type,omitempty" json:"type,omitempty"`
+	VunerableBelow string `bson:"vulnerablebelow,omitempty" json:"vulnerablebelow,omitempty"`
+	Version        string `bson:"version,omitempty" json:"version,omitempty"`
+	Occurrences    int    `bson:"occurrences,omitempty" json:"occurrences,omitempty"`
+}
+
+// HuskyCIResults is a struct that represents huskyCI scan results.
+type HuskyCIResults struct {
+	GoResults         GoResults         `bson:"goresults,omitempty" json:"goresults,omitempty"`
+	PythonResults     PythonResults     `bson:"pythonresults,omitempty" json:"pythonresults,omitempty"`
+	JavaScriptResults JavaScriptResults `bson:"javascriptresults,omitempty" json:"javascriptresults,omitempty"`
+	RubyResults       RubyResults       `bson:"rubyresults,omitempty" json:"rubyresults,omitempty"`
+}
+
+// GoResults represents all Golang security tests results.
+type GoResults struct {
+	HuskyCIGosecOutput HuskyCISecurityTestOutput `bson:"gosecoutput,omitempty" json:"gosecoutput,omitempty"`
+}
+
+// PythonResults represents all Python security tests results.
+type PythonResults struct {
+	HuskyCIBanditOutput HuskyCISecurityTestOutput `bson:"banditoutput,omitempty" json:"banditoutput,omitempty"`
+	HuskyCISafetyOutput HuskyCISecurityTestOutput `bson:"safetyoutput,omitempty" json:"safetyoutput,omitempty"`
+}
+
+// JavaScriptResults represents all JavaScript security tests results.
+type JavaScriptResults struct {
+	HuskyCIRetireJSOutput HuskyCISecurityTestOutput `bson:"retirejsoutput,omitempty" json:"retirejsoutput,omitempty"`
+	HuskyCINpmAuditOutput HuskyCISecurityTestOutput `bson:"npmauditoutput,omitempty" json:"npmauditoutput,omitempty"`
+}
+
+// RubyResults represents all Ruby security tests results.
+type RubyResults struct {
+	HuskyCIBrakemanOutput HuskyCISecurityTestOutput `bson:"brakemanoutput,omitempty" json:"brakemanoutput,omitempty"`
+}
+
+// HuskyCISecurityTestOutput stores all Low, Medium and High vulnerabilities for a sec test
+type HuskyCISecurityTestOutput struct {
```

Comment:
The "Qualitative Severity Rating Scale" at [CVSS 3.0](https://www.first.org/cvss/v3.0/specification-document#5-Qualitative-Severity-Rating-Scale) has 4 principal scores.
```
Low	0.1 - 3.9
Medium	4.0 - 6.9
High	7.0 - 8.9
Critical	9.0 - 10.0
```
This can serve as a basis for HusckyCI's number of scores.

Reply:
Nice! I will work on this soon here at #316! Thanks! 🙃